mooniefill.lic

```diff
@@ -0,0 +1,86 @@
+custom_require.call(%w[common common-moonmage])
+
+class MoonieFill
+
+  def initialize
+    @magic = ['Dawgolesh', 'Durgaulda', 'Ismenia','Yavash', 'Wolf', 'Cobra', 'Ox', 'Donkey', 'Coyote', 'Dove', 'Phoenix', 'Cow', 'Shrike', 'Jackal', 'Amlothi', 'Wren', 'Nightingale', 'Owl', 'Adder', 'Shrew', 'Toad']
+    @lore = ['Cobra', 'Donkey', 'Wren', 'Ismenia', 'Xibar', 'Raven', 'Dolphin', 'Albatross', 'Heron', 'Welkin', 'Phoenix', 'Shardstar', 'Weasel', 'King Snake', 'Hare', 'Brigantine', 'Scales', 'Amlothi', 'Verena']
+    @survival = ['Sun', 'Dolphin', 'Brigantine', 'Ox', 'Heart', 'Lion', 'Unicorn', 'Boar', 'Panther', 'Wolverine', 'Shark', 'Goshawk', 'Raccoon', 'Triquetra', 'Szeldia', 'Heron', 'Cow', 'Shrike', 'Ram', 'Vulture', 'Archer']
+    @offense = ['Szeldia', 'Er\'qutra', 'Boar', 'Panther', 'Wolverine', 'Shark', 'Triquetra', 'Weasel', 'Scales', 'Coyote', 'Owl', 'Shrew', 'Cat', 'Mongoose', 'Merewalda', 'Archer', 'Adder', 'Scorpion', 'Centaur', 'Viper', 'Spider']
+    @defense = ['Merewalda', 'Dawgolesh', 'Mongoose', 'Scorpion', 'Centaur', 'Lion', 'Goshawk', 'Raccoon', 'Morleena', 'Albatross', 'Welkin', 'Shardstar', 'King Snake', 'Hare', 'Dove', 'Nightingale', 'Katamba', 'Magpie', 'Vulture', 'Jackal', 'Giant']
+    @pools = ['Survival', 'Magic', 'Lore', 'Defense', 'Offense']
+    do_the_things
+  end
+
+
+  def fill_pools
+    @pools.each do |pool|
+      case DRC.bput("predict state #{pool}", /no understanding/, /feeble understanding/, 
```

Comment:
`astrology.lic` does something similar that we might want to commonize:

https://github.com/rpherbig/dr-scripts/blob/acc89c9a453fcb5afe06e4fa36e31e8b7b8bd4dd/astrology.lic#L105